virtualization/ostep-05-process-api/src/main.rs

```diff
@@ -1,3 +1,20 @@
+use std::process;
+
+extern {
+    fn fork() -> u32;
+    fn getpid() -> u32;
+}
+
 fn main() {
-    println!("put some stuff here");
+    unsafe {
```

Comment:
You can avoid using unsafe here and also benefit from the type system by using the nix bindings.

See: https://docs.rs/nix/0.15.0/nix

Here is fork specifically: https://docs.rs/nix/0.15.0/src/nix/unistd.rs.html#225